src/scripts/expletives.js

```diff
@@ -19,7 +19,7 @@ const expletives = {
 	bich: "plinimal",
 	cock: "phloopdie",
 	slut: "phliminustrim",
-  	whore: "phlungus",
+	whore: "phlungus",
```

Comment:
Not too sure what the difference is here, VSCode's auto-formatting did it so I assume this is how it should be